install_config/install/host_preparation.adoc

```diff
@@ -547,6 +547,95 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[enabling-image-signature-support]]
+=== Enabling Image Signature Support
+
+{product-title} is capable of cryptographically verifying images are from
+trusted sources. The
+xref:../../security/deployment.adoc#security-deployment-from-where-images-deployed[Container Security Guide] provides a high-level description of how image signing works.
+
+Configuration is performed by the `atomic` command line interface (CLI), version
+1.12.5 or greater.
+ifdef::openshift-enterprise[]
+It is pre-installed on RHEL Atomic Host systems.
+
+[NOTE]
+====
+For more on the `atomic` CLI, see the
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/cli_reference/prerequisites[Atomic CLI documentation].
+====
+endif::[]
+
+Install the *atomic* package if it is not installed on the host system:
+
+----
+$ yum install atomic
+----
+
+The **atomic trust** sub-command manages trust configuration. The default
+configuration is to whitelist all registries. This means no signature
+verification is configured.
+
+----
+$ atomic trust show
+* (default)                         accept
+----
+
+A reasonable configuration might be to whitelist a particular registry or
+namespace, blacklist (reject) untrusted registries, and require signature
+verification on a vendor registry. The following set of commands performs this
+example configuration:
+
+.Example Atomic Trust Configuration
+----
+$ atomic trust add --type insecureAcceptAnything 172.30.1.1:5000
+
+$ atomic trust add --sigstoretype atomic \
+  --pubkeys pub@example.com \
+  172.30.1.1:5000/production
+  
+$ atomic trust add --sigstoretype atomic \
+  --pubkeys /etc/pki/example.com.pub \
+  172.30.1.1:5000/production
+  
+$ atomic trust add --sigstoretype web \
+  --sigstore https://access.redhat.com/webassets/docker/content/sigstore \
+  --pubkeys /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release \
+  registry.access.redhat.com
+  
+$ sudo atomic trust show
+* (default)                         accept
+172.30.1.1:5000                     accept
+172.30.1.1:5000/production          signed security@example.com
+registry.access.redhat.com          signed security@redhat.com,security@redhat.com
+----
+
+When all the signed sources are verified, nodes may be further hardened with a
+global `reject` default:
+
+----
+$ atomic trust default reject
+
+$ atomic trust show
+* (default)                         reject
+172.30.1.1:5000                     accept
+172.30.1.1:5000/production          signed security@example.com
+registry.access.redhat.com          signed security@redhat.com,security@redhat.com
+----
+
+Use the `atomic` man page `man atomic-trust` for additional examples.
```

Comment:
Should *atomic* be bold in the first reference as a descriptor of man page? 
Like *atomic-openshift-master* service